src/nodedef.cpp

```diff
@@ -633,7 +635,14 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 		visual_solidness = 1;
 		break;
 	case NDT_ALLFACES_OPTIONAL:
-		if (tsettings.leaves_style == LEAVES_FANCY) {
+		if (tsettings.leaves_style == LEAVES_PLANTLIKE) {
+			drawtype = NDT_PLANTLIKE;
+			solidness = 0;
+			visual_scale = 1.2;
```

Comment:
I guess this is to make plantlike horizontally fill the diagonal, but textures will vertically extend beyond node edges, and Z-fight with other leaves, so not sure about this.